Makefile

```diff
@@ -29,3 +32,87 @@ go/test:
 .PHONY: release
 release: go/test
 	goreleaser build
+
+start_opensearch: create_opensearch wait_for_opensearch
+
+create_opensearch:
+	${CONTAINER_RUNTIME} network create opensearch_network
+	${CONTAINER_RUNTIME} run -d --name=opensearch -p 9200:9200 -p 9600:9600 -e "discovery.type=single-node" -e "plugins.security.disabled=true" opensearchproject/opensearch:${OPENSEARCH_VERSION}
+	${CONTAINER_RUNTIME} run -d --name=opensearch_dashboards -p 5601:5601 -e "OPENSEARCH_HOSTS=[\"http://opensearch:9200\"]" -e "DISABLE_SECURITY_DASHBOARDS_PLUGIN=true" opensearchproject/opensearch-dashboards:${OPENSEARCH_VERSION}
+	${CONTAINER_RUNTIME} network connect opensearch_network opensearch
+	${CONTAINER_RUNTIME} network connect opensearch_network opensearch_dashboards
+
+remove_opensearch:
+	${CONTAINER_RUNTIME} stop opensearch
+	${CONTAINER_RUNTIME} rm opensearch
+	${CONTAINER_RUNTIME} stop opensearch_dashboards
+	${CONTAINER_RUNTIME} rm opensearch_dashboards
+	${CONTAINER_RUNTIME} network rm opensearch_network
+
+restart_opensearch: remove_opensearch start_opensearch
+
+# init_smoke_test always re-initializes the .terraform folder even if it is already present.
+init_smoke_test:
+	cd smoketest && rm -rf .terraform && rm -f .terraform.lock.hcl && terraform init
+smoke_test: init_smoke_test smoke_test_fast
+
+# this only creates the .terraform folder if it is not already present
+smoketest/.terraform:
```

Comment:
Am I missing something or do `init_smoke_test` and `smoketest/.terraform` do the exact same?

Reply:
smoketest/.terraform only creates the folder if it doesn't yet exist (otherwise the makefile automatically skips the step because the file is present and not forced with PHONY). With the command init_smoke_test a re-creation of the folder is forced even if it already exists which makes the smoketest more stable because it is always created from scratch.
But for the smoke_test_fast command (less stable but faster) I still wanted to add the option which only generates if the folder doesn't exist.
Does that make sense?

Reply:
Yes, that does make sense. Still I'm wondering if we could invoke `init_smoke_test` from the `smoketest/.terraform` target, to avoid the code duplication. 

I was  thinking either making it a dependency, e.g.
```makefile
smoketest/.terraform: init_smoke_test
```

or invoking it via make
```makefile
smoketest/.terraform:
   $(MAKE) init_smoke_test
```

Reply:
yeah I like the second option 👍 PR has been updated